### DISCUSSION TITLE
Playtime: Clip Legato Mode

### DISCUSSION THREAD

**Discussion by helgoboss** [helgoboss/helgobox#1043] [Ideas]

**Comment by ThoughtPhotography** [comment]
**Legato mode** is pretty critical so you can launch clips at any time and have them inherit the relative playback position from the previous clip which maintains synchronization. This came up frequently when I asked a group of Live user's what they thought were the most important Session View features.

> Suppose you have gathered, in one track, a number of looping clips, and you now want to toggle among them without losing the sync. For this you could use a large quantization setting (one bar or greater), however, this might limit your musical expression.
> 
> Another option, which works even with quantization turned off, is to engage Legato Mode for the respective clips. When a clip in Legato Mode is launched, it takes over the play position from whatever clip was played in that track before. Hence, you can toggle clips at any moment and rate without ever losing the sync.
> 
> Legato Mode is very useful for creating breaks, as you can momentarily play alternative loops and jump back to what was playing in the track before.
> 
> Unless all the clips involved play the same sample (differing by clip settings only), you might hear dropouts when launching clips in Legato Mode. This happens because you are unexpectedly jumping to a point in the sample that Live has had no chance to pre-load from disk in advance. You can remedy this situation by engaging Clip RAM Mode (see [‘Clip RAM Mode’](https://www.ableton.com/en/live-manual/12/clip-view/#clip-ram-mode)) for the clips in question.

https://www.ableton.com/en/live-manual/12/launching-clips/#legato-mode
